Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -0,0 +1,157 @@
+import SwiftUI
+
+/// Showing two choices from which only one can be selected.
+///
+/// Currently SegmentedSwitch allows to have 2 segments.
+/// SegmentedSwitch can be in error state only in unselected state.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/segmentedswitch/)
+public struct SegmentedSwitch<Value>: View where Value: LosslessStringConvertible & Equatable {
+
+    @Binding private var selection: Value?
+    let label: String
+    let _message: Message?
+    let firstOption: Value
+    let secondOption: Value
+
+    let separatorWidth: CGFloat = .xxxSmall
+    let borderWidth: CGFloat = .xxxSmall
```

Comment:
Should probably be equal to `BorderWidth.emphasis` to match other components

Reply:
Alright, didn't know about this enum. Took 2pt value from Figma

Reply:
We seem to have a mismatch there - will be fixed in a future follow-up.

Reply:
For now used `BorderWidth.emphasis` as was suggested for both `borderWidth` and `separatorWidth` constants. No other values are used